src/main/java/com/uber/jenkins/phabricator/conduit/DifferentialClient.java

```diff
@@ -114,8 +114,22 @@ public JSONObject postComment(String message) throws IOException, InterruptedExc
     }
 
     protected JSONObject callConduit(String methodName, Map<String, String> params) throws IOException, InterruptedException, ArcanistUsageException {
-        ArcanistClient arc = new ArcanistClient(this.arcPath, "call-conduit", params, this.conduitToken, methodName);
+        ArcanistClient arc = getArcanistClient(methodName, params);
         return arc.parseConduit(this.launcher.launch(), this.launcher.getStderr());
```

Comment:
also for a future diff -- this should just be `launcher.lanch()` instead of `this.launcher.launch()`, right?